acceptance/setup/git/pre-suite/000_EnvSetup.rb

```diff
@@ -12,6 +12,7 @@
     'git',
     'ruby',
     'rubygem-json',
+    'rubygem-io-console'
```

Comment:
* Should `rubygem-io-console` already be baked into images? Thoughts @jcoconnor?
* Is `rubygem-io-console` available on all `:redhat` (RHEL) based distros?  I believe we're mostly concerned here with Fedora, but there will be some variation across different RHEL versions and the Ruby versions that they want to install by default as `ruby`